Dockerfile

```diff
@@ -1,14 +1,7 @@
-FROM node:8.1.0
+FROM python:2.7.13
 
-ENV SERVERLESS_VERSION 1.18.0
-
-RUN npm install serverless@${SERVERLESS_VERSION} -g
-
-RUN apt-get update
```

Comment:
Should the update stay in the image?

Reply:
Yes, but this is unneeded now.  Below this, the installation of `node` triggers the same update, so the end result is the same as doing it manually.

Reply:
:+1: